tests/copilot_helper/test_command_dns.py

```diff
@@ -739,3 +743,298 @@ def test_get_required_subdomains_throws_exception_with_multiple_errors():
     message = exc_info.value.args[0]
     assert f"Subdomain {subdomain} is not a valid domain" in message
     assert f"{subdomain} is not a subdomain of {base_domain}" in message
+
+
+@mock_sts
+@mock_elbv2
+@mock_ec2
+@mock_ecs
+@mock_acm
+def test_cdn_add_if_domain_already_exists(alias_session, aws_credentials):
```

Comment:
At a glance it looks like there is a lot of shared stuff in the setup of the new tests. These could be extracted out to helper methods for readability and less duplication.

Reply:
removed duplication to make more readable